.vscode/cspell.json

```diff
@@ -8,7 +8,8 @@
   ],
   "ignorePaths": [
     "**/tests/recordings/**",
-    ".vscode/cspell.json"
+    ".vscode/cspell.json",
+    "eng/**/*.json"
```

Comment:
This glob looks good. You'll need to rebase and pick up the changes to cspell.json from Friday